nerfstudio/scripts/exporter.py

```diff
@@ -59,6 +59,30 @@ class Exporter:
     """Path to the output directory."""
 
 
+def validate_pipeline(normal_method, normal_output_name, pipeline: Pipeline) -> None:
```

Comment:
Add types and documentation for the args.

Reply:
np, I just updated that file